drivers/shared/executor/executor_linux.go

```diff
@@ -355,6 +366,9 @@ func (l *LibcontainerExecutor) Shutdown(signal string, grace time.Duration) erro
 	case <-time.After(time.Second * 15):
 		return fmt.Errorf("process failed to exit after 15 seconds")
 	}
+
+	l.iowg.Wait()
```

Comment:
Should this be covered by the 15 second timeout above?  Also, in the universal executor case, it'd be nice to have a grace period to copy output before killing processes.

Reply:
I'm curious about the grace period to copy output. My understanding is that using a pipe, it will get closed when the child process exits, so all logs will already be inflight through the pipe and block Wait until they're copied?